src/viam/utils.py

```diff
@@ -190,7 +190,7 @@ def sensor_readings_native_to_value(readings: Mapping[str, Any]) -> Mapping[str,
     return {key: primitive_to_value(value) for (key, value) in prim_readings.items()}
 
 
-def sensor_readings_value_to_native(readings: Mapping[str, Value]) -> Mapping[str, Any]:
+def sensor_readings_value_to_native(readings: Mapping[str, Value]) -> Mapping[str, ValueTypes]:
```

Comment:
I see now why we had it as `Any`: `Vector3`, `GeoPoint`, and `Orientation` are not `ValueTypes`. So this new signature would not pass type checking. (It'll still work because python doesn't actually care about types.)

To make this return type more accurate, I suggest we create a new type alias: `SensorReading`, which is a `Union[ValueTypes, Orientation, GeoPoint, Vector3]`. So the final return type would be: `Mapping[str, SensorReading]`.

This way, if we ever support other sensor reading types (`EulerAngles`, `Quaternion`, etc.), we only have to update it here, and not for each method signature.